docs/libs/gelly_guide.md

```diff
@@ -240,6 +240,7 @@ Graph<Long, Double, Double> networkWithWeights = network.joinWithEdgesOnSource(v
     <img alt="Union Transformation" width="50%" src="fig/gelly-union.png"/>
 </p>
 
+* <strong>Difference</strong>: Gelly's `difference()` method performs a difference on the vertex and edge sets of the input graphs. The resultant graph is formed by removing the common vertices and edges from the graph.
```

Comment:
I think @vasia also wanted you to update the description for union ^^

Now, this still looks a bit unclear. It seems that there are two input graphs. You should make it obvious that the current graph gets differentiated with an input graph. That way, you won't leave room for comments.